src/phoenix/server/api/queries.py

```diff
@@ -78,6 +79,49 @@
 
 @strawberry.type
 class Query:
+    @strawberry.field
+    async def model_providers(
+        self, vendors: List[str], info: Info[Context, None]
+    ) -> List[ModelProvider]:
+        all_vendors = {
+            "OpenAI": ModelProvider(  # https://platform.openai.com/docs/models
+                name="OpenAI",  # currently only models using the chat completions API
```

Comment:
since this will have to be a graphql input param, it might make more sense to start with the vendor being a graphql enum